app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,20 @@
  *= require_tree .
  *= require_self
  */
+body {
+}
+ .home {
+  max-height: 100vh;
+  min-height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #e6e6e6;
+  padding-bottom: 50px;
+ }
+
+ .navbar-collapse {
+  align-items: center;
+    justify-content: center;
+ }
+
```

Comment:
It's preferable to create separated files for the styles, e.g. one CSS file for home styles, one CSS file for navbar, one CSS file for the footer, etc. By this way, you have all the styles well organized and separated from each other.

In your application.css file you can only add styles to reset the  predefined styles by the browser.